event-management-backend/src/test/java/ro/msg/event_management/SaveEventIntegrationTest.java

```diff
@@ -65,27 +81,21 @@ void testSaveEvent() {
                 .ticketsPerUser(10)
                 .highlighted(false)
                 .maxPeople(30)
-                .startDate(LocalDate.now())
-                .endDate(LocalDate.now())
-                .startHour(LocalTime.now())
-                .endHour(LocalTime.now().plusHours(3))
+                .startDate(LocalDate.parse("2020-11-11"))
+                .endDate(LocalDate.parse("2020-11-15"))
+                .startHour(LocalTime.parse("18:00"))
+                .endHour(LocalTime.parse("20:00"))
                 .creator("")
                 .picturesUrlSave(picturesUrlSave)
                 .ticketCategoryDtoList(ticketCategoryDtoList)
                 .location(1)
```

Comment:
What would this 1 point to?
<details>
  <summary>Spoiler warning</summary>
This location does not seem to be used in the tested logic. Either remove it or point to the created location.
</details>

Reply:
It was useless because the service get the location Id for insert. I removed it.